beeline/__init__.py

```diff
@@ -427,6 +427,24 @@ def finish_span(span):
     if _GBL:
         _GBL.tracer_impl.finish_span(span=span)
 
+def marshal_trace_context():
+    '''
+    Returns a serialized form of the current trace context (including the trace
+    id and the current span), encoded as a string. You can use this to propagate
+    trace context to other services.
+
+    Example:
+
+    ```
+    trace_context = beeline.marshal_trace_context()
+    headers = {'X-Honeycomb-Trace'}
```

Comment:
Are you supposed to add `trace_context` to the `X-Honeycomb-Trace` header somehow? I would like this example to show what to do with `trace_context`.

Reply:
Oops, good catch